backend/utils.py

```diff
@@ -18,32 +18,37 @@
 ###################
 ###### Utils ######
 ###################
+from http import HTTPStatus
```

Comment:
here we import a new module from the Python standard library to use the Enum for http statuses instead of hardcoding the return codes